assembly-multiuser/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/multiuser.properties

```diff
@@ -120,3 +124,18 @@ che.keycloak.github.endpoint=NULL
 
 #     The number of seconds to tolerate for clock skew when verifying exp or nbf claims.
 che.keycloak.allowed_clock_skew_sec=3
+
+#     Use the OIDC optional `nonce` feature to increase security.
+che.keycloak.use_nonce=true
```

Comment:
shall we provide some implementation agnostic name such as  ```che.openid.use_nonce``` - ?

Reply:
Well, for now this idea was to do as few modifications as possible.

Additionally, it's an option related to the *Keycloak Javascript Adapter* client library.
So having it called `che.keycloak.use_nonce` seems consistent to me.

We are still using the Keycloak Javascript adapter, but I simply made it more friendly to act as client of alternate OIDC servers.